Anlab.Mvc/Controllers/LabController.cs

```diff
@@ -201,7 +213,7 @@ public IActionResult ListUsersOrders(string id)
             return View(model);
         }
         [HttpPost]
-        public async Task<IActionResult> UpdateFromCompletedTests(int id, bool confirm, IFormFile uploadFile)
+        public async Task<IActionResult> UpdateFromCompletedTests(int id, bool confirm, IFormFile uploadFile, string labComments, decimal adjustmentAmount)

```

Comment:
@laholstege Not sure how easy it would be but if you could put this into an UpdateFromCompletedTestsModel.cs or something and have the method only take the (id, model) as params and let model binding take care of the rest, that would be a lot cleaner.